src/components/comments/Comments.jsx

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import "./comments.css";
+
+const Comments = () => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { article_Id } = useParams();
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://hstestncnews.herokuapp.com/api/articles/${article_Id}/comments`
+      )
+      .then(({ data }) => {
+        setComments(data.article);
+        setIsLoading(false);
+      });
+  }, [article_Id]);
+
+  if (isLoading) return <h2>Loading...</h2>;
+
+  return (
+    <div>
+      <div className="commentSection">
+        <h2>Comments</h2>
+        <div className="allComments">
+          {comments.map((comment) => (
```

Comment:
Everything in a .map() can be abstracted out into its own component, so why don't you put this in a `CommentCard.jsx` or something similar?

Reply:
I was thinking that as I was doing it (similar to the articleCard, but thought since I'm only using it in one page, best to keep it there. I'll refactor 👍